src/components/Card/indexCard.js

```diff
@@ -0,0 +1,61 @@
+import Image from 'next/image';
+
+import useStore from '../../hooks/useStore';
+import StyledCalendarButton from '../Buttons/CalendarButton/styled';
+import StyledCardBody from '../CardBody/styled';
+import StyledCardDescription from '../CardDescription/styled';
+import CardFooter from '../CardFooter';
+import StyledCardHeader from '../CardHeader/styled';
+import StyledCardHeadline from '../CardHeadline/styled';
+
+import StyledCard from './styled';
+
+export default function IndexCard({card}) {
+	const addToPlanner = useStore(state => state.addToPlanner);
+	const changeSets = useStore(state => state.changeSets);
+	const changeReps = useStore(state => state.changeReps);
+	const changeWeight = useStore(state => state.changeWeight);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+	}
+	function handleSetInput(input) {
+		changeSets(card.id, Number(input));
+	}
+
+	function handleRepInput(input) {
+		changeReps(card.id, Number(input));
+	}
+
+	function handleWeightInput(input) {
+		changeWeight(card.id, Number(input));
+	}
+	return (
+		<StyledCard>
+			<StyledCardHeader>
+				<StyledCardHeadline>{card.name}</StyledCardHeadline>
+				<StyledCalendarButton
+					onClick={() => {
+						addToPlanner(card.id);
+					}}
+				>
+					{'add to planner'}
```

Comment:
You don't need the `{ }`

```suggestion
					add to planner
```